### PR TITLE
Create ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by bode

Ticket: [DM-1214](https://cumulocity.atlassian.net/browse/DM-1214)
Original PR: [6373](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6373)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: User message in device list page is incorrect when no devices are available in the tenant [GRAFT][release/cd]
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Improved user message on device list page when no devices are available
```

File: content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-1214
+version: 1019.24.3
+---
+User message in device list page is incorrect when no devices are available in the tenant [GRAFT][release/cd]
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, when no devices were available in a tenant, the device list page displayed an incorrect user message. With this change, the user message on the device list page has been updated to provide a clear and accurate message when no devices are available in the tenant. This improvement enhances the user experience by providing precise feedback on the device availability status.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md:17]:
```suggestion
The layout of the empty state message displayed in data grids across all standard applications has been adjusted and unified. Grid that did not differentiate between no data available and no filter matches or search results are now adjusted to display the corresponding empty state message.

For developers the `emptyStateContext` directive was introduced that injects data source statistics variable into template context.
```

File: content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-1214
+version: 1019.24.3
+---
+The layout of the empty state message displayed in data grids across all standard applications has been adjusted and unified. Grid that did not differentiate between no data available and no filter matches or search results are now adjusted to display the corresponding empty state message.
```
Comment (dafkk) [inline content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md:17]:
```suggestion
The layout of the empty state message displayed in data grids across all standard applications has been adjusted and unified. Grids that did not differentiate between no data available and no filter matches or search results are now adjusted to display the corresponding empty state message.
```

**Review by dafkk — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-1214
+version: 1019.24.3
+---
+The layout of the empty state message displayed in data grids across all standard applications has been adjusted and unified. Grids that did not differentiate between no data available and no filter matches or search results are now adjusted to display the corresponding empty state message.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md:17]: @bozhidardedov Is it only the layout that has changed or also the message itself?

Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md:17]:
```suggestion
The layout of the empty state message displayed in data grids across all standard applications has been adjusted and unified. Moreover, grids that did not differentiate between no data available and no filter matches or search results do now display a more precise corresponding empty state message.
```

File: content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+---
+date: ""
+title: Improved user message on Device list page when no devices are available
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-24-3-user-message-in-device-list-page-is-incorrect-when-no-devices.md:3]:
```suggestion
title: Improved layout and content of empty state messages
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**